jekyll/_cci2/using-macos.md

```diff
@@ -35,10 +22,28 @@ jobs:
       - run: xcodebuild -version
 ```
 
-### macOS VM Storage
+## Supported Xcode versions
+{: #supported-xcode-versions }
+
+{% include snippets/xcode-versions.md %}
+
+## Available resource classes
+{: #available-resource-classes }
+
+{% include snippets/macos-resource-table.md %}
+
+```yaml
+jobs:
+  build:
+    macos:
+      xcode: 12.5.1
+    resource_class: large
+```
+
+## macOS VM Storage
 {: #macos-vm-storage }
 
-The amount of available storage on our macOS VMs depends on the resource class and Xcode image being used. The size of the Xcode images varies based on which tools are pre-installed.
+The amount of available storage on CircleCI macOS virtual machines depends on the resource class and Xcode image being used. The size of the Xcode images varies based on which tools are pre-installed. The table below indicates how much storage will be available with various Xcode/resource class combinations. Also note the exceptions to this noted below.
```

Comment:
FYI I made some changes to this intro to the table, and to the phrasing of the note to try and clear up the exceptions